javascript/script.js

```diff
@@ -0,0 +1,73 @@
+// for mobile menu
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+const body = document.querySelector('body');
+// for speakers
+const speakersCards = document.querySelector('.instructors-list');
+// speakers end
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  body.classList.toggle('active');
+});
+document.querySelectorAll('.nav-options').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+  body.classList.remove('active');
+}));
+// mobile menu end
+// for speakers
+const SpeakersList = [
+  {
+    speakername: 'Satya Nadella',
+    imagelink: '../images/yochai.jpg',
+    desc1: 'Microsoft CEO ',
+    desc2: 'Satya Nadella knows a thing or two about success in business. The technology executive reportedly earned over $16 million in pay last year',
+  },
+  {
+    speakername: 'Bill Gates',
+    imagelink: '../images/sohyoeng.jpg',
+    desc1: 'Co-founder of Microsoft',
+    desc2: 'William Henry Gates III is an American business magnate, software developer, investor, author, and philanthropist. He is a co-founder of Microsoft',
+  },
```

Comment:
## Dynamically generated HTML

- [ ] Good job on generating the section "Featured speakers" dynamically with JavaScript. 👍 👍 however, it would be better to provide a valid path of the speaker's image to make it visible to your page 😂
- [ ] Kindy to replicate the original design, it would be better to use only **two** columns on you desktop version 👍  

**Your design**
![image](https://user-images.githubusercontent.com/102757126/210803574-5110b692-3fd8-4f7d-a7b8-0f9ce913cc73.png)

**Original design**
![image](https://user-images.githubusercontent.com/102757126/210803917-ecde32f9-113e-4764-8cb4-1aef46b9c06a.png)